.circleci/config.yml

```diff
@@ -41,12 +44,31 @@ jobs:
       xcode-version: "12.4.0"
     steps:
       - setup_environment
+      - copy_demo_credentials
       - run:
           name: Build App Store Target
           command: bundle exec fastlane build_app_store
 
+  Upload to App Store:
+    executor:
+      name: ios/default
+      xcode-version: "12.4.0"
+    steps:
+      - setup_environment
+      - run:
+          name: Decrypt Secrets
+          command: bundle exec fastlane run configure_apply
+      - run:
+          name: Install Sentry CLI
+          command: brew install getsentry/tools/sentry-cli
+      - run:
+          name: Build and Upload for App Store
+          command: bundle exec fastlane build_and_upload_app_store create_github_release:true
```

Comment:
What do you think of having to explicitly pass `true` here?

The other projects have this logic, too, only the value is computed at runtime based on the kind of tag and on the `is_ci` return value.

Reply:
I actually find this (passing explicitly like here) better personally. Though in the end it's nbd.